openassessment/xblock/test/test_staff_area.py

```diff
@@ -746,7 +747,7 @@ def mock_formatter(entries):
         )
 
         # Enable teams
-        xblock.teams_enabled = True
+        is_team_assignment_patch.return_value = True
```

Comment:
If you wait until I merge my PR, you could rebase and use the `_setup_xblock_and_create_submission` function to simplify a bunch of this (https://github.com/edx/edx-ora2/pull/1412)

Reply:
I could also use `get_team_for_anonymous_user` instead of duplicating that work. Will wait until after #1412

Reply:
Merged!